### DISCUSSION TITLE
[B 站无关水] VSC 扩展 JavaScript and TypeScript Nightly > v5.4.x 可能会导致 .vue 文件中自定义组件代码高亮不正确

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4801] [综合讨论 / General]
怪哦

不正确显示：
![图片](https://github.com/the1812/Bilibili-Evolved/assets/81268244/067620ec-eebe-4b6f-a85e-646b0e56b946)

正确显示：
![图片](https://github.com/the1812/Bilibili-Evolved/assets/81268244/2b5ef35b-957b-4af3-a211-18f4f1f1e4e9)

**Comment by the1812** [comment]
volar 奇奇怪怪的问题一直不少(
